spec/features/add_new_peep.rb

```diff
@@ -0,0 +1,19 @@
+require 'peep'
+
+feature 'User adds a new peep' do
+  scenario 'when browsing the homepage' do
+    expect(Peep.count).to eq(0)
+    visit '/'
+    add_peep('Hello this is my first peep')
+    expect(Peep.count).to eq(1)
+    peep = Peep.first
+    expect(peep.message).to eq('Hello this is my first peep')
+  end
+
+  def add_peep(message)
```

Comment:
Unused method argument - `message`. If it's necessary, use `_` or `_message` as an argument name to indicate that it won't be used. You can also write as `add_peep(*)` if you want the method to accept any arguments but don't care about them.